<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionPreference"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <widget class="QMenu" name="menuDraw">
     <property name="title">
      <string>Draw</string>
     </property>
     <addaction name="action2DLayout"/>
    </widget>
    <widget class="QMenu" name="menuParaxial">
     <property name="title">
      <string>Paraxial</string>
     </property>
     <addaction name="actionFirstOrderData"/>
     <addaction name="actionParaxialRayTrace"/>
    </widget>
    <widget class="QMenu" name="menuRaysAndSpot">
     <property name="title">
      <string>Rays and Spot</string>
     </property>
     <addaction name="actionSingleRayTrace"/>
     <addaction name="separator"/>
     <addaction name="actionSpotDiagram"/>
     <addaction name="actionThroughfocusSpotDiagram"/>
    </widget>
    <widget class="QMenu" name="menuAberration">
     <property name="title">
      <string>Aberration</string>
     </property>
     <addaction name="actionRayAberration"/>
     <addaction name="actionOpticalPath"/>
     <addaction name="separator"/>
     <addaction name="actionFieldCurvature"/>
     <addaction name="actionGridDistortion"/>
     <addaction name="actionLongitudinalAberration"/>
     <addaction name="actionLateralColor"/>
     <addaction name="actionChromaticFocusShift"/>
     <addaction name="separator"/>
     <addaction name="actionSeidelCoefficients"/>
    </widget>
    <addaction name="actionPrescription"/>
    <addaction name="separator"/>
    <addaction name="menuDraw"/>
    <addaction name="separator"/>
    <addaction name="menuParaxial"/>
    <addaction name="separator"/>
    <addaction name="menuRaysAndSpot"/>
    <addaction name="menuAberration"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionDebugStream"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSetVigFactors"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Geopter</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="text">
    <string>Preference</string>
   </property>
  </action>
  <action name="actionDebugStream">
   <property name="text">
    <string>Debug Stream</string>
   </property>
  </action>
  <action name="action2DLayout">
   <property name="text">
    <string>2D Layout</string>
   </property>
  </action>
  <action name="actionSetVigFactors">
   <property name="text">
    <string>Set Vignetting Factors</string>
   </property>
  </action>
  <action name="actionFirstOrderData">
   <property name="text">
    <string>First Order Data</string>
   </property>
  </action>
  <action name="actionParaxialRayTrace">
   <property name="text">
    <string>Paraxial Trace</string>
   </property>
  </action>
  <action name="actionSingleRayTrace">
   <property name="text">
    <string>Single Ray Trace</string>
   </property>
  </action>
  <action name="actionSpotDiagram">
   <property name="text">
    <string>Standard Spot Diagram</string>
   </property>
  </action>
  <action name="actionThroughfocusSpotDiagram">
   <property name="text">
    <string>Throughfocus Spot Diagram</string>
   </property>
  </action>
  <action name="actionRayAberration">
   <property name="text">
    <string>Ray Aberration</string>
   </property>
  </action>
  <action name="actionFieldCurvature">
   <property name="text">
    <string>Field Curvature</string>
   </property>
  </action>
  <action name="actionLongitudinalAberration">
   <property name="text">
    <string>Longitudinal Aberration</string>
   </property>
  </action>
  <action name="actionOpticalPath">
   <property name="text">
    <string>Optical Path</string>
   </property>
  </action>
  <action name="actionGridDistortion">
   <property name="text">
    <string>Grid Distortion</string>
   </property>
  </action>
  <action name="actionLateralColor">
   <property name="text">
    <string>Lateral Color</string>
   </property>
  </action>
  <action name="actionChromaticFocusShift">
   <property name="text">
    <string>Chromatic Focus Shift</string>
   </property>
  </action>
  <action name="actionSeidelCoefficients">
   <property name="text">
    <string>Seidel Coefficients</string>
   </property>
  </action>
  <action name="actionPrescription">
   <property name="text">
    <string>Prescription</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
